zombienet/launch-rococo-local-with-integritee-xcm.toml

```diff
@@ -0,0 +1,67 @@
+[settings]
+timeout = 10000
+
+[relaychain]
+chain = "rococo-local"
+default_command = "../../bin/polkadot-v0.9.27"
+default_args = [ "-lparachain=debug" ]
+
+[[relaychain.nodes]]
+name = "alice"
+validator = true
+ws_port = 9999
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+ws_port = 9988
+
+[[relaychain.nodes]]
+name = "charlie"
+validator = true
+ws_port = 9977
+
+[[relaychain.nodes]]
+name = "eve"
+validator = true
+ws_port = 9966
+
+[[parachains]]
+id = 2015
+addToGenesis = true
+cumulus_based = true
+chain = "integritee-rococo-local-dev"
+
+[[parachains.collators]]
+name = "integritee-collator01"
+command = "./target/release/integritee-collator"
+ws_port = 9994
+
+[[parachains.collators]]
+name = "integritee-collator02"
+command = "./target/release/integritee-collator"
+ws_port = 9995
+
+#this is not working !!!
```

Comment:
Please make an issue with details of the problem and refer to the issue from here.

Reply:
With zombienet release [v1.2.73](https://github.com/paritytech/zombienet/releases/tag/v1.2.73), I managed to launch a acala/karura parachain. 
I updated the config to open HRMP channles according to [issue](https://github.com/paritytech/zombienet/issues/449)